actionmailer/lib/action_mailer/message_delivery.rb

```diff
@@ -138,14 +138,36 @@ def enqueue_delivery(delivery_method, options = {})
         else
           job = @mailer_class.delivery_job
 
-          if job <= MailDeliveryJob
+          if use_legacy_args?(job)
             job.set(options).perform_later(
               @mailer_class.name, @action.to_s, delivery_method.to_s, args: @args)
+          elsif job <= DeliveryJob
+            job.set(options).perform_later(
+              @mailer_class.name, @action.to_s, delivery_method.to_s, *@args)
           else
+            ActiveSupport::Deprecation.warn(<<~EOM)
+              In Rails 6.2, Action Mailer will pass the mail arguments inside the `:args` kwargs.
```

Comment:
@Edouard-chin Can we spell out `kwargs` here to say `keyword argument`? I am sure not everyone understands it or I would like to consider at least 10 people don't and deprecation message should be easy to understand. What do you think?

Reply:
Sounds good 👍